upscale_swin2sr/README.md

```diff
@@ -0,0 +1 @@
+Run `upscale_images.py` to upscale the images of [timbrooks/instructpix2pix-clip-filtered](https://huggingface.co/datasets/timbrooks/instructpix2pix-clip-filtered). The upscaler is Swin2SR from [this repository](https://github.com/mv-lab/swin2sr). `swin2sr` is entirely from the original repository. `upscale_image.py` is a modified version of `predict.py` from [here](https://github.com/mv-lab/swin2sr/blob/main/predict.py). 
```

Comment:
No need to review the contents of `swin2sr` folder as it comes directly from the original repository: https://github.com/mv-lab/swin2sr/.